src/azure-cli/azure/cli/command_modules/identity/_help.py

```diff
@@ -28,3 +30,53 @@
 type: command
 short-summary: Lists available operations for the Managed Identity provider
 """
+
+helps['identity federated-credential'] = """
+type: group
+short-summary: Manage federated identity credentials under user assigned identities.
+"""
+
+helps['identity federated-credential create'] = """
+type: command
+short-summary: Create a federated identity credential under an existing user assigned identity.
+examples:
+  - name: Create a federated identity credential under a specific user assigned identity.
+    text: |
+        az identity federated-credential create --name myFicName --identity-name myIdentityName --resource-group myResourceGroup --issuer myIssuer --subject mySubject --audiences myAudiences
+"""
```

Comment:
This command `az identity federated-credential create` acts on **managed identity**: https://learn.microsoft.com/en-us/azure/active-directory/workload-identities/workload-identity-federation-create-trust-user-assigned-managed-identity?pivots=identity-wif-mi-methods-azcli

`az ad app federated-credential create` (#23122) acts on **AAD application**: https://learn.microsoft.com/en-us/azure/active-directory/workload-identities/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azcli

However, they expose different command interfaces:

- `az identity federated-credential create` uses separated arguments `--issuer`, `--subject` and `--audiences`
- `az ad app federated-credential create` uses `--parameters` which includes `issuer`, `subject` and `audiences` in a JSON payload. This decision was made in email thread: _federatedIdentityCredentials's implementation in Azure CLI_

In my opinion, this causes inconsistency and increases users' learning cost.